lib/msf/ui/console/command_dispatcher/auxiliary.rb

```diff
@@ -89,8 +89,7 @@ def name
   # Tab completion for the run command
   #
   def cmd_run_tabs(str, words)
-    return [] if words.length > 1
-    @@auxiliary_opts.fmt.keys
```

Comment:
Looks like the auxiliary options have been lost here 👀

Reply:
I'll get that added back in.